File: source/api/customers/update/_update_simple.py
import omise

omise.api_secret = 'skey_test_4xs8breq3htbkj03d2x'

customer = omise.Customer.retrieve('cust_test_4xtrb759599jsxlhkrb')
customer.update(
  email='john.smith@example.com',
  description='Another description'
)

# Or alternatively:
customer.email = 'john.smith@example.com'
customer.description = 'Another description'
customer.update()

File: source/api/customers/update/_update_simple.cs
var customerUpdateInfo = new CustomerInfo ();
customerUpdateInfo.Id = "cust_test_4xtrb759599jsxlhkrb";
customerUpdateInfo.Email = "john.smith@example.com";
customerUpdateInfo.Description = "Another description";

var updateResult = client.CustomerService.UpdateCustomer(customerUpdateInfo);
